examples/web_app.py

```diff
@@ -0,0 +1,20 @@
+"""
+Example for running Application using the `aiohttp.web` CLI.
+
+Run this app using::
+
+    $ python -m aiohttp.web web_app.init
+"""
+
+from aiohttp.web import Application, Response
+
+
+def hello_world(req):
+    return Response(text="Hello World")
+
+
+def init(args):
```

Comment:
Example should have a code for demonstrating how to parse `args` again via `argparse`.
I mean `parse.parse_args(args)` -- not every user know about the ability.
I want to encourage argparse usage, not ugly code like `if args[0] == '--param' and args[1] == 'value'`.